notes/src/View.js

```diff
@@ -3,61 +3,87 @@ import './index.css';
+      color                 : "lightgray",
+      display               :  "none"
+  };
+
+  const customStyles = {
+      top                   : '50%',
+      left                  : '50%',
+      right                 : 'auto',
+      bottom                : 'auto',
+      marginRight           : '-50%',
+      transform             : 'translate(-50%, -50%)',
+      color                 : "lightgray",
+      display               :  "block"
+  };
+  
+  
+
 class View extends Component{
+    constructor(){
+        super();
+        this.state = {
+            display : customStylesHidden
+        }
+    }
 
     componentDidMount(){
+        console.log("View","componentDidMount")
         this.props.getNotes();
     }
 
-    render() {
-
-            const id = this.props.match.params.id;
-
-            const test =  this.props.notes.find(note => id === note._id)
-
-
-            const {0:jsxObj=null} =  this.props.notes.filter(note => id === note._id)
-            .map( e => {
-                const note = <Note 
-                key={e._id} 
-                name={e._id}
-                title={e.title}
-                textBody={e.textBody}
-                _id={e._id}/>
-
-                const form = <Form 
-                key={e._id} 
-                name={e._id}
-                title={e.title}
-                textBody={e.textBody}
-                _id={e._id}/>
-                return {note:note,form:form}
-                }
-                )
-
+    deleteHandler = (event,input) => {
+        // this.setState({
+        //     display:customStyles
+        // })
+        event.preventDefault();
+        this.props.history.push('/')
+        this.props.deleteNote(input)
+        this.props.getNotes()
+    }
 
-                const [note,form] = (jsxObj ? [jsxObj.note, jsxObj.form] : [null,null])
-                // const url = (note ? `/note/${note.props._id}/edit` : "#" )
-            if(!this.props.edit){
 
-                return(
 
-                    <>       
-                        <Link to={`/note/${test._id}/edit`}>Edit</Link>
-                        <br />
-                        <Link to="#">Delete</Link>
-                        {note}
-                    </>
+    render() {
+        console.log("View","Render")
+        if(this.props.notes.length===0){
+            return(
+                <p>Loading...</p>
+            )
+        } else {
+                const id = this.props.match.params.id;
+                const note =  this.props.notes.find(note => id === note._id)
+    
+                if(!this.props.edit){
+                    return(
+    
+                        <>       
+                            <Link to={`/note/${note._id}/edit`}>Edit</Link>
+                            <br />
+                            <Link to="#" onClick={(event,id = note._id) => this.deleteHandler(event,id)}>Delete</Link>
+                            <Note id={note._id} title={note.title} textBody={note.textBody}/>
```

Comment:
I like that you're trying to make the Note component reusable, but the styles are pretty different! Try to match the style guidelines and don't worry about using a reusable component here, this View is your reusable component in this case!